.env.sample

```diff
@@ -11,3 +11,6 @@ SESSION_SECRET=# secret key for session ID generation
 
 # Successful authentication redirect URL
 HOMEPAGE_URL=# url where user will be redirected after successful authentication
+
+# Whitelist of domains
+DOMAINS =# list of domains, eg - http://example.com,https://google.com
```

Comment:
Whitelist for what? Can you make these more realistic? Like localhost and the editor.